applications/GeoMechanicsApplication/custom_strategies/schemes/newmark_dynamic_U_Pw_scheme.hpp

```diff
@@ -289,6 +289,30 @@ class NewmarkDynamicUPwScheme : public NewmarkQuasistaticUPwScheme<TSparseSpace,
         KRATOS_CATCH( "" )
     }
 
+    void CalculateRHSContribution(
+        Condition& rCurrentCondition,
+        LocalSystemVectorType& RHS_Contribution,
+        Element::EquationIdVectorType& EquationId,
+        const ProcessInfo& CurrentProcessInfo) override
```

Comment:
Once more the `r` prefix.
```suggestion
        const ProcessInfo& rCurrentProcessInfo) override
```